website/src/components/UseAnalytics.tsx

```diff
@@ -0,0 +1,20 @@
+import ReactGA from 'react-ga4'
+
+type UseAnalyticsFunction = {
+  init: (trackingId: string) => void;
+  trackPageViewed: (path: string) => void;
+}
+
+export const useAnalytics = () : UseAnalyticsFunction => {
```

Comment:
is there a need to have a `useAnalytics` method? Could we just do:

```ts
export function init (...) { ... }
export function trackPageViewed (...) { ... }
```
?

Then maybe import it like that:

```ts
import { init, trackPageViewed } from '../utils/analytics'
```